docs/src/references/mqtt-api.md

```diff
@@ -231,24 +231,25 @@ graph LR
 
-|Segment|Description|
-|-------|-----------|
-|&lt;category&gt;|Category of data (telemetry or commands) which is related to the **identifier**. A fixed list of categories is used to represent data types such as; measurements, alarms, events, commands etc.|
-|&lt;type&gt;|A unique type/name used to identify the information being pushed or received. Types allow users to filter/subscribe to data which interests them. For example, a measurement could be published under a type called "flow_rate", then other clients interested in the flow rate can subscribe to that single typed topic.|
-|`...`|Additional channel specific topic segments. Each **category** is responsible for defining the number and meaning of the remaining topic segments.|
+| Segment          | Description                                                                                                                                                                                                                                                                                                               |
+|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| &lt;category&gt; | Category of data (telemetry or commands) which is related to the **identifier**. A fixed list of categories is used to represent data types such as; measurements, alarms, events, commands etc.                                                                                                                          |
+| &lt;type&gt;     | A unique type/name used to identify the information being pushed or received. Types allow users to filter/subscribe to data which interests them. For example, a measurement could be published under a type called "flow_rate", then other clients interested in the flow rate can subscribe to that single typed topic. |
+| `...`            | Additional channel specific topic segments. Each **category** is responsible for defining the number and meaning of the remaining topic segments.                                                                                                                                                                         |
 
 
 #### Categories
 
 The following is an overview of the channel categories which are available.
 
-|Category|Description|
-|--------|-------|
-|m|Measurements|
-|e|Events|
-|a|Alarms|
-|cmd|Commands|
-|twin|Entity twin metadata|
+| Category | Description          |
+|----------|----------------------|
+| m        | Measurements         |
+| e        | Events               |
+| a        | Alarms               |
+| cmd      | Commands             |
+| twin     | Entity twin metadata |
+| status   | Device status        |
```

Comment:
Here the `status` is service status right?

Reply:
fixed 3b2953620f